common/scala/src/main/scala/whisk/http/BasicHttpService.scala

```diff
@@ -19,56 +19,64 @@ package whisk.http
 
 import scala.concurrent.duration.DurationInt
 import scala.language.postfixOps
+import scala.concurrent.Await
+import scala.collection.immutable.Seq
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
As a general rule neither the implicit context should be used or an explicit Await. @dubeejw can you scrub all occurrences that might have been introduced in the code (tests are different).